CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Move high cardinality values into structured-metadata:
+  - kubernetes audit log `resource` label
+  - `filename` label
+  - log `output stream` label
+- Rename `node_name` label into `node` to match metric label.
```

Comment:
Is the node_name to node renaming really needed ? I would rather not doing it unless we have a really good reason for it.

Reply:
Yes we do it to align with the metric label as was decided in the refinement last week https://github.com/giantswarm/giantswarm/issues/30308#issuecomment-2355694791

Reply:
Ok fine